openweather/src/main/java/driver/DriverGetter.java

```diff
@@ -46,4 +50,10 @@ private static ChromeOptions initChromeOptions() {
 
         return options;
     }
+
+    private static FirefoxOptions initFirefoxOptions () {
+        FirefoxOptions options = new FirefoxOptions();
+        options.addPreference("geo.enabled", false);
```

Comment:
what is this used for? do we have any feature that requires to have geo enabled ?